scripts/ffmpeg_automator.py

```diff
@@ -15,9 +15,12 @@ def get_audio_maps(streams):  # noqa: WPS231
     audio_languages = json.loads(os.environ['AUDIO_LANGUAGES'])
 
     get_first_audio_per_lang_only = os.environ['FIRST_AUDIO_PER_LANG_ONLY'].lower() == 'true'
+    use_highest_channels = os.environ['HIGHEST_CHANNELS'].lower() == 'true'
     lang_found = []
     index = -1
 
+    # Dictionary to keep track of the highest channel count stream per language (only if HIGHEST_CHANNELS is true)
+    highest_channels_per_lang = {} if use_highest_channels else None
```

Comment:
```suggestion
    if use_highest_channels:
        highest_channels_per_lang = {}
```
else None isn't needed